libr/asm/p/asm_arm_cs.c

```diff
@@ -113,7 +175,21 @@ static int disassemble(RAsm *a, RAsmOp *op, const ut8 *buf, int len) {
 	}
 	if (op && !op->size) {
 		op->size = insn->size;
-		char *buf_asm = sdb_fmt ("%s%s%s",
+		if (insn->id == ARM_INS_IT) {
+			disass_itblock (a, insn);
+		}
+		char *buf_asm;
+		ut64 itcond;
+		itcond = ht_uu_find (ht_it,  a->pc, &found);
+		if (found){
+			insn->detail->arm.cc = itcond;
+			insn->detail->arm.update_flags = 0;
+			r_str_cpy(insn->mnemonic, 
+				r_str_newf ("%s%s",
```

Comment:
if you allocate this here on the fly you can't free it, so we will leak a lot of memory.

Reply:
see latest commit